### DISCUSSION TITLE
Reverse Proxy

### DISCUSSION THREAD

**Discussion by jeremehancock** [EmulatorJS/EmulatorJS#405] [Q&A]
First let me say this is amazing!! I am loving this so much!

I am self-hosting this using the Docker container from linuxserver. I would like to put it behind a reverse proxy. I know how to do all of that, but my concern is with the File manager thing on the top left of the UI. Does that allow connection to the server? Since there isn't authentication for this I am concerned that would be insecure. I know I could secure the reverse proxy in other ways, I just wanted to understand more about how the file manager connects to the back end. 

Thanks

**Comment by ethanaobrien** [comment]
Very sorry for the delay, I've been quite busy.

You probably want to open this in the linuxserver repository. His docker emulatorjs project actually doesn't use the frontend or backend from this project (which makes the name emulatorjs confusing). Very sorry about this, I hope this can help. If there's something you think I could help with please let me know

**Comment by jeremehancock** [comment]
No worries at all. Sorry about that. It is very confusing especially since the logos are the same. 😃

**Comment by mpclepto** [comment]
Did you ever get an answer to this? I just blackhole the filebrowser.html (as seen in a reddit comment elsewhere) like this:

In NPM, set a custom location for it

/filebrowser.html
And and send it to http 127.0.0.1 80
